src/main/java/de/kekru/struktogrammeditor/control/Controlling.java

```diff
@@ -293,16 +297,12 @@ public void actionPerformed(ActionEvent e) {
 			Helpers.openWebsite("http://strukt.whiledo.de/changelog.html");
 			break;
 
-		case kontaktformular:
-			Helpers.openWebsite("http://strukt.whiledo.de/kontakt.php");
-			break;
-
 		case hilfe:
 			Helpers.openWebsite("http://strukt.whiledo.de/hilfe.html");
 			break;
 
 		case sourceCode:
-			Helpers.openWebsite("https://github.com/kekru/struktogrammeditor/tree/" + GlobalSettings.buildInfoGitHash);

```

Comment:
Please revert this. I want to jump to the correct sources :)  
Be sure to revert changes im pom.xml to make it work

Or tell me, why it is not useful ^^

Reply:
It was a request from Matthias Arndt, I applied it but will revert if you say so